lang-python/python-libclang/spec

```diff
@@ -0,0 +1,4 @@
+VER=18.1.1
```

Comment:
To me this package has a low quality.  It ships a copy of libclang.so.18.1, which is outdated and IMO doing so is just bloating the system.  Could the pymupdf package be updated to use https://pypi.org/project/clang/ instead?